### PR TITLE
    IPLDT Scan Fixes

### PR DESCRIPTION
**Pull request by petertix**
- Changed references to be: Intel® Manager for Lustre* software
     - Changed references to be: Intel Corporation
     - The official names can be found here: namesdb.intel.com
     - Re-scanned the repo and ensured that the TM errors went away.

---
This change is [Reviewable](https://reviewable.io/reviews/intel-hpdd/intel-manager-for-lustre/305)

### CODE REVIEW

**Comment by johnsonw** [comment]
I will load this up on my cluster and check the registered symbols.

**Comment by johnsonw** [comment]
```
  File "setup.py", line 22
SyntaxError: Non-ASCII character '\xc2' in file setup.py on line 22, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
make[1]: *** [tarball] Error 1
make[1]: Leaving directory `/var/lib/jenkins/workspace/manager-for-lustre/arch/x86_64/distro/el7/chroma-agent'
make: *** [chroma-agent] Error 2
Build step 'Execute shell' marked build as failure
```

**Comment by petertix** [comment]
Good catch Will.

**Comment by jgrund** [comment]
>   File "setup.py", line 22
SyntaxError: Non-ASCII character '\xc2' in file setup.py on line 22, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
make[1]: *** [tarball] Error 1
make[1]: Leaving directory `/var/lib/jenkins/workspace/manager-for-lustre/arch/x86_64/distro/el7/chroma-agent'
make: *** [chroma-agent] Error 2
Build step 'Execute shell' marked build as failure

Thanks for confirming, that was what I was wondering.

**Comment by petertix** [comment]
I loaded this build on my vagrant adm node successfully.
I clicked on the "Admin" tab and at the bottom, the footer shows:

Intel® Manager for Lustre* software Build jenkins_manager_for_lustre_1581 is Copyright © 2017 Intel Corporation. All rights reserved. About Intel® Manager for Lustre* software System Status

So far, so good. Still clicking and testing.

**Comment by jgrund** [comment]
Just in case it's not clear, not all help strings come from the IML repo. There are a bunch from other repos as well (In case you are testing this patch expecting that).

**Comment by petertix** [comment]
Thanks. That makes sense. I was just checking to see if anything was obviously broken and so far I am not seeing anything from the initial install.

File: chroma-diagnostics/setup.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
     version = package_version(),
     packages = find_packages(),
     include_package_data = True,
-    author = "Intel(R) Corporation",
+    author = "Intel Corporation",
```
Comment (johnsonw) [inline chroma-diagnostics/setup.py]: Does this not need the register symbol?

Comment (petertix) [reply chroma-diagnostics/setup.py]: According to namesdb.intel.com:

The approved usage is: Intel Corporation

This name has been approved and adopted as a tradename.
No noun or trademark symbol (such as, ®, ™ or SM) should be used with this name.

**Pull request merged**